core/src/main/scala/com/broadinstitute/dsp/DbReaderImplicits.scala

```diff
@@ -16,6 +16,7 @@ object DbReaderImplicits {
   )
 
   implicit val gcsBucketNameGet: Get[GcsBucketName] = Get[String].map(GcsBucketName)
+  implicit val initBucketNameGet: Get[InitBucketName] = Get[String].map(s => InitBucketName(s.trim.subSequence(5, s.trim.length - 1).toString))
```

Comment:
maybe use `temap` here? and fail if this isn't in the format of `gs://???/`

Reply:
Yes, it does. I added a require in the most recent round of refactors I'm about to push